modules/social_features/social_group/config/install/flag.flag.mute_group_notify.yml

```diff
@@ -0,0 +1,30 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - group
+id: mute_group_notify
+label: 'Mute Group Notify'
+bundles: {  }
+entity_type: group
+global: false
+weight: 0
+flag_short: 'Mute notify'
+flag_long: 'Muting the group notifications will result in their notifications not being send you anymore.'
+flag_message: ''
+unflag_short: 'Unmute notify'
```

Comment:
'Unmute'

Reply:
Moritz suggested calls it a "Unmute group". So I renamed it "Unmute group".